### PR TITLE
TW 386 Update instructions for Scratch Pad export

### PR DESCRIPTION
**Pull request by HeatherStenson**
* Updates instructions for exporting Scratch Pad data - since users are logged out, data dumps are downloaded directly rather than being emailed to them.

### CODE REVIEW

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 1]
```diff
@@ -41,22 +41,22 @@ To leave the Scratch Pad:
...
 
-When you leave the Scratch Pad, your data remains there; you won't see it in your workspace. But you can export the data from the Scratch Pad, then import it to a workspace.
+When you leave the Scratch Pad, your data remains there; you won't see it in your workspace. But you can export the data from the Scratch Pad as JSON, then import it to a workspace.
 
-To export all Scratch Pad data:
+To export saved Scratch Pad data:
 
 1. Go to the Scratch Pad.
 1. Select the gear icon in the upper-right corner to open **Settings**, and select the **Data** tab.
 1. Click **Export Data**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/using-scratch-pad.md:50]: Found 4 instances of "click" on this page > chance to "select"

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 2]
```diff
@@ -41,22 +41,22 @@ To leave the Scratch Pad:
...
-1. This opens the **Export data** page in your web browser. Click **Export Data**.
-1. Select **Collections**, **Environments**, or both, and select **Request Data Export**.
-1. You'll get an email with a link to download the data dump, which is a single JSON file containing your data.
+1. In the save dialog, choose a directory in which to save the file, then select **Save**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/using-scratch-pad.md:51]: Either use "dialog box" or just delete "in the save dialog," altogether as it's not really needed

**Review by mark-anthony-postman — APPROVED** [review]
Couple of minor comments, otherwise LGTM! 🙂

File: src/pages/docs/getting-started/using-scratch-pad.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/using-scratch-pad.md]: Ayayay! Thanks!

File: src/pages/docs/getting-started/using-scratch-pad.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/using-scratch-pad.md]: Good call - updated!

**Pull request merged**